network/socket-bench-server/src/main.rs

```diff
@@ -34,13 +34,20 @@ fn main() {
         .unwrap();
     let executor = rt.handle();
 
+    // passed as TCP_ADDR=/ip6/::1/tcp/12345
     if let Some(addr) = args.tcp_addr {
         let addr = start_stream_server(&executor, TcpTransport::default(), addr);
         info!("bench: tcp: listening on: {}", addr);
     }
 
+    // passed as TCP_NOISE_ADDR=/ip4/1.2.3.4/tcp/1234
     if let Some(addr) = args.tcp_noise_addr {
-        let addr = start_stream_server(&executor, build_tcp_noise_transport(), addr);
+        let (noise_transport, public_key) = build_tcp_noise_transport();
+        let addr = start_stream_server(&executor, noise_transport, addr);
+
+        // display full networkaddr
+        let handshake_version = 0;
+        let addr = addr.append_prod_protos(public_key, handshake_version);
```

Comment:
oops, you only want to append the `/ln-noise-ik` protocol, since these bench Transports are not running the `/ln-handshake` protocol.

Reply:
ah, well I removed this, I'll add this to another PR